ddpui/management/commands/set_uuids_in_orgtask_deployment.py

```
@@ -0,0 +1,206 @@
+        for dataflow in OrgDataFlowv1.objects.filter(org=org).all():
+            logger.info(
+                f"processing dataflow with deployment_id {dataflow.deployment_id}"
+            )
+            # fetch deployment params
+            deployment = None
+            try:
+                deployment = prefect_service.get_deployment(dataflow.deployment_id)
+            except Exception as error:
+                logger.info(
+                    f"Something went wrong in fetching the deployment with id '{dataflow.deployment_id}'"
+                )
+                logger.exception(error)
+                logger.info("skipping to next loop")
+                continue
+
+            # add uuid to each task config in params
+            params = deployment["parameters"]
+            if "config" in params and "tasks" in params["config"]:
+                for task_config in params["config"]["tasks"]:
+                    if "slug" in task_config:
+                        org_task = OrgTask.objects.filter(
+                            org=org, task__slug=task_config["slug"]
+                        )
+                        if "connection_id" in task_config:
+                            org_task = org_task.filter(
+                                connection_id=task_config["connection_id"]
+                            )
+
+                        org_task = org_task.first()
+
+                        if org_task is not None and org_task.uuid:
+                            task_config["orgtask_uuid"] = str(org_task.uuid)
+                            logger.info(
+                                f"updated task config with uuid for {task_config['slug']}"
+                            )
+                logger.info(
+                    f"updated deployment params for {dataflow.name}|{dataflow.deployment_id}"
+                )
+            else:
+                logger.info(
+                    f"no tasks key found in deployment params for {dataflow.deployment_id}"
```

we upgraded all of them right? so over here you can raise an exception